package.json

```diff
@@ -21,39 +21,47 @@
     "registry": "https://registry.npmjs.org"
   },
   "dependencies": {
-    "@hapi/good": "8.2.1",
-    "@hapi/hapi": "18.3.2",
-    "@mojaloop/central-services-shared": "7.4.8",
-    "@mojaloop/central-services-error-handling": "7.4.2",
+    "@hapi/good": "8.2.3",
+    "@hapi/hapi": "18.4.0",
+    "@mojaloop/central-services-error-handling": "7.5.0",
+    "@mojaloop/central-services-logger": "8.1.1",
+    "@mojaloop/central-services-shared": "8.1.2",
     "axios": "0.19.0",
-    "blipp": "4.0.0",
+    "blipp": "4.0.1",
     "good-console": "8.0.0",
     "good-squeeze": "5.1.0",
     "hapi-openapi": "1.2.2",
     "json-rules-engine": "4.0.0",
-    "knex": "0.19.3",
+    "knex": "0.19.4",
     "memory-cache": "0.2.0",
     "mysql": "2.17.1",
     "node-fetch": "2.6.0",
     "rc": "1.2.8"
   },
   "devDependencies": {
-    "eslint": "6.3.0",
-    "istanbul": "1.1.0-alpha.1",
-    "npm-audit-resolver": "1.5.0",
+    "eslint": "6.4.0",
+    "npm-audit-resolver": "2.0.0",
+    "npm-check-updates": "3.1.23",
+    "nyc": "14.1.1",
+    "pre-commit": "1.2.2",
     "proxyquire": "2.1.3",
-    "sinon": "7.4.1",
+    "sinon": "7.5.0",
```

Comment:
please add standard dependency and add appropriate script and pre-commit

Reply:
Done